examples/cp/tension_single_xml.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+import sys
+sys.path.append('../..')
+
+import numpy as np
+
+from neml.cp import singlecrystal
+from neml.math import rotations, tensors, nemlmath
+from neml import parse
+
+import matplotlib.pyplot as plt
+
+if __name__ == "__main__":
+  L = np.array([[-0.5,0,0],[0,1.0,0],[0,0,-0.5]])
+  erate = 1.0e-4
+  steps = 25
+  emax = 0.5
+
+  E = 100000.0
+  nu = 0.3
+
+  t0 = 50.0
+  ts = 50.0
+  b = 100.0
```

Comment:
The results between the tension_single.py ans tension_single_xml.py are different, is that deliberate (due to inclusion of orientation ?), also the value of b in the xml file is 10, should it be 100 to be consistent ?

Reply:
They weren't supposed to be the same.

Reply:
okay got it.